0.19-SNAPSHOT (kogito-cleanup) crashed with RuntimeException while running J2CL on BUNDLE Mode

It's not possible to run or build j2cl app (reactor or simple app) using BUNDLE mode. 

in ADVANCED mode it works as expected.

```
INFO] Starting org.treblereel:j2cl-tests:1.0-SNAPSHOT/optimized_js
java.io.FileNotFoundException: /Users/treblereel/workspace/j2cl/j2cl-tests/target/gwt3BuildCache/0.19-SNAPSHOT/org.treblereel-j2cl-tests-1.0-SNAPSHOT/4580de8955ee77b23b773cb5abafdaef-optimized_js/results/j2cl-tests/j2cl-tests.js (No such file or directory)
        at java.base/java.io.FileOutputStream.open0(Native Method)
        at java.base/java.io.FileOutputStream.open(FileOutputStream.java:298)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:237)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:126)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.filenameToOutputStream(AbstractCommandLineRunner.java:1935)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.fileNameToOutputWriter2(AbstractCommandLineRunner.java:1926)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.outputManifestOrBundle(AbstractCommandLineRunner.java:2195)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.outputBundle(AbstractCommandLineRunner.java:2153)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.processResults(AbstractCommandLineRunner.java:1565)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.doRun(AbstractCommandLineRunner.java:1324)
        at com.google.javascript.jscomp.AbstractCommandLineRunner.run(AbstractCommandLineRunner.java:535)
        at com.vertispan.j2cl.tools.Closure.compile(Closure.java:183)
        at com.vertispan.j2cl.build.provided.ClosureTask$2.execute(ClosureTask.java:134)
        at com.vertispan.j2cl.build.TaskScheduler$2.executeTask(TaskScheduler.java:172)
        at com.vertispan.j2cl.build.TaskScheduler$2.lambda$onReady$0(TaskScheduler.java:211)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
[ERROR] Exception executing task org.treblereel:j2cl-tests:1.0-SNAPSHOT/optimized_js
java.lang.IllegalStateException: Closure Compiler failed, check log for details
    at com.vertispan.j2cl.build.provided.ClosureTask$2.execute (ClosureTask.java:154)
    at com.vertispan.j2cl.build.TaskScheduler$2.executeTask (TaskScheduler.java:172)
    at com.vertispan.j2cl.build.TaskScheduler$2.lambda$onReady$0 (TaskScheduler.java:211)
    at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:515)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run (ScheduledThreadPoolExecutor.java:304)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:829)
java.lang.RuntimeException
        at com.vertispan.j2cl.build.DiskCache.markFailed(DiskCache.java:452)
        at com.vertispan.j2cl.build.DiskCache$CacheResult.markFailure(DiskCache.java:54)
        at com.vertispan.j2cl.build.TaskScheduler$2.executeTask(TaskScheduler.java:181)
        at com.vertispan.j2cl.build.TaskScheduler$2.lambda$onReady$0(TaskScheduler.java:211)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.ut
        
```

it can be reproduced by kogito-editors-java or https://github.com/treblereel/j2cl-tests

--- a/j2cl-maven-plugin/src/it/hello-world-single/invoker.properties
+++ b/j2cl-maven-plugin/src/it/hello-world-single/invoker.properties
@@ -1,0 +1,7 @@
+# As a very simple project, this is a good one to try each optimization level
+invoker.goals.1 = verify -DcompilationLevel=ADVANCED_OPTIMIZATIONS
+invoker.goals.2 = verify -DcompilationLevel=BUNDLE
+invoker.goals.3 = verify -DcompilationLevel=WHITESPACE_ONLY
+invoker.goals.4 = verify -DcompilationLevel=SIMPLE_OPTIMIZATIONS
+
+invoker.goals.5 = verify -DcompilationLevel=BUNDLE_JAR

--- a/j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/ClosureTask.java
+++ b/j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/ClosureTask.java
@@ -93,6 +93,7 @@ public class ClosureTask extends TaskFactory {
                 // set up a source directory to build from, and to make sourcemaps work
                 // TODO move logic to the "post" phase to decide whether or not to copy the sourcemap dir
                 String jsOutputDir = new File(closureOutputDir + "/" + initialScriptFilename).getParent();
+                Files.createDirectories(Paths.get(jsOutputDir));
                 final File sources;
                 final Map<String, List<String>> js;
                 if (!sourcemapsEnabled) {

